package.json

```diff
@@ -15,13 +17,14 @@
     "@types/jasmine": "^3.5.0",
     "@types/lodash": "^4.14.149",
     "@types/request": "2.48.4",
+    "dtslint": "^2.0.3",
     "jasmine": "^3.5.0",
-    "mocha": "^6.2.2"
+    "mocha": "^6.2.2",
+    "npm-run-all": "^4.1.5"
   },
   "dependencies": {
     "commander": "^4.0.1",
     "dot": "^1.1.3",
-    "dtslint": "^2.0.3",
```

Comment:
Thinking about this, should we just make _all_ dependencies dev given this project is not really intended to be pulled in via a package manager?

Reply:
Well, maybe we will publish it to npm someday, so that users could generate fresh typings using `npx google-api-typings-generator ...`. 
I'm not sure, does `npx` pull `devDependencies`? I don't think so.

Reply:
> I'm not sure, does `npx` pull `devDependencies`?

Yeah, if this project was installed as another project's dependency, `devDependencies` would be excluded. I'm still not sure I see the use case, but I'm fine leaving the rest of these as-is.